pydoctor/epydoc2stan.py

```diff
@@ -139,11 +139,11 @@ def resolve_identifier_xref(self, identifier: str, lineno: int) -> str:
 
         if identifier == fullID:
             self.obj.report(
-                "invalid ref to '%s' not resolved" % (identifier,),
+                "invalid ref to \"%s\" not resolved" % (identifier,),
```

Comment:
not sure if you want single or double quoted in the human readable output.

I prefer double quotes for human readable as they stand out and are easier to read.

Reply:
Double quotes is fine with me.

I do prefer to single-quote the string in that case, since backslashes draw a lot of unnecessary attention when reading the code. Even if we choose to adopt Black at some point, it won't convert string literals if that leads to more backslashes.

This kind of thing is subjective though, so if you have a strong preference for escaped double quotes, keep them.

Reply:
I will revert to single quotes. Thanks for the feedback.